ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -26,8 +27,10 @@ const emptyStorage: VulnMgmtLocalStorage = {
 
 function WorkloadCvesOverviewPage() {
     const [storedValue, setStoredValue] = useLocalStorage('vulnerabilityManagement', emptyStorage);
+    const { setPage } = useURLPagination(20);
```

Comment:
This feels a little clunky, since it sets a default page size both at the top level here as well as when this is used in child components further down the tree. It doesn't have an impact now, but the multiple calls could get out of sync as the code changes. The alternative would be to prop drill the `pagination` object down from here to the necessary child components.

This prop drilling is currently happening to a lesser extent and being passed a single level from three hook calls:
```
CVEsTableContainer (useURLPagination)
|- TableEntityToolbar

ImagesTableContainer (useURLPagination)
|- TableEntityToolbar
 
DeploymentsTableContainer (useURLPagination)
|- TableEntityToolbar
```
and if we moved this to the top level it would instead be passed three levels from one hook call:
```
WorkloadCVEsOverviewPage (useURLPagination)
|- CVEStatusTabNavigation

|-|- CVEsTableContainer
|-|-|- TableEntityToolbar

|-|- ImagesTableContainer
|-|-|- TableEntityToolbar

|-|- CVEsTableContainer
|-|-|- TableEntityToolbar
```

I vaguely feel like having multiple components in a component tree writing state to the same URL parameter is asking for trouble, which this PR would introduce. Any strong opinions here, or fine to leave it as-is?

Reply:
hm i do feel like that having multiples of the same hook at different levels is not exactly ideal, but at least we're storing the state in the URL -- i think as long as the URL stays the ultimate source of truth, we should be able to minimize the negative effects for the most part. if anything, removing the `CVEStatusTabNavigation` component should have lessened the nesting, so i think it makes sense to revise it to just be passed from one hook call, either here or in a future PR.

Reply:
Ah good call, I'll go ahead and update this now so we don't have to worry about coming back to it.